src/test/java/picard/analysis/CollectRnaSeqMetricsTest.java

```diff
@@ -112,6 +115,59 @@ public void testBasic() throws Exception {
         Assert.assertEquals(metrics.PCT_R2_TRANSCRIPT_STRAND_READS, 0.666667);
     }
 
+    @DataProvider(name = "rRnaIntervalsFiles")
+    public static Object[][] rRnaIntervalsFiles() throws IOException {
+        return new Object[][] {
+                {null},
+                {File.createTempFile("tmp.rRna.", ".interval_list")}
+        };
+    }
+
+    @Test(dataProvider = "rRnaIntervalsFiles", expectedExceptions = PicardException.class)
+    public void testNoIntevalsNoFragPercentage(final File rRnaIntervalsFile) throws Exception {
+        final SAMRecordSetBuilder builder = new SAMRecordSetBuilder(true, SAMFileHeader.SortOrder.coordinate);
+
+        // Add a header but no intervals
+        if ( rRnaIntervalsFile != null  ) {
+            final IntervalList rRnaIntervalList = new IntervalList(builder.getHeader());
+            rRnaIntervalList.write(rRnaIntervalsFile);
+            rRnaIntervalsFile.deleteOnExit();
+        }
+
+        // Create some alignments
+        final String sequence = "chr1";
+        final String ignoredSequence = "chrM";
+
+        // Set seed so that strandedness is consistent among runs.
+        builder.setRandomSeed(0);
+        final int sequenceIndex = builder.getHeader().getSequenceIndex(sequence);
+        builder.addPair("pair1", sequenceIndex, 45, 475);
+        builder.addFrag("ignoredFrag", builder.getHeader().getSequenceIndex(ignoredSequence), 1, false);
+
+        final File samFile = File.createTempFile("tmp.collectRnaSeqMetrics.", ".sam");
+        samFile.deleteOnExit();
+
+        final SAMFileWriter samWriter = new SAMFileWriterFactory().makeSAMWriter(builder.getHeader(), false, samFile);
+        for (final SAMRecord rec : builder.getRecords()) samWriter.addAlignment(rec);
+        samWriter.close();
+
+        // Generate the metrics.
+        final File metricsFile = File.createTempFile("tmp.", ".rna_metrics");
+        metricsFile.deleteOnExit();
+
+        final String rRnaIntervalsPath = rRnaIntervalsFile != null ? rRnaIntervalsFile.getAbsolutePath() : null;
+        final String[] args = new String[]{
+                "INPUT=" + samFile.getAbsolutePath(),
+                "OUTPUT=" + metricsFile.getAbsolutePath(),
+                "REF_FLAT=" + getRefFlatFile(sequence).getAbsolutePath(),
+                "RIBOSOMAL_INTERVALS=" + rRnaIntervalsPath,
+                "RRNA_FRAGMENT_PERCENTAGE=" + 0.0,
+                "STRAND_SPECIFICITY=SECOND_READ_TRANSCRIPTION_STRAND",
+                "IGNORE_SEQUENCE=" + ignoredSequence
+        };
+        Assert.assertEquals(runPicardCommandLine(args), 0);
+    }
```

Comment:
There's a test where the intervals file has a header but no intervals and `RRNA_FRAGMENT_PERCENTAGE=0`. An empty intervals file will fail regardless of the value of `RRNA_FRAGMENT_PERCENTAGE`:
```
java.lang.IllegalStateException: Interval list file must contain header. 

at htsjdk.samtools.util.IntervalList.fromReader(IntervalList.java:455)
at htsjdk.samtools.util.IntervalList.fromFile(IntervalList.java:400)
at picard.analysis.directed.RnaSeqMetricsCollector.makeOverlapDetector(RnaSeqMetricsCollector.java:69)
at picard.analysis.CollectRnaSeqMetrics.setup(CollectRnaSeqMetrics.java:155)
at picard.analysis.SinglePassSamProgram.makeItSo(SinglePassSamProgram.java:122)
at picard.analysis.SinglePassSamProgram.doWork(SinglePassSamProgram.java:77)
at picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:205)
at picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:94)
at picard.cmdline.PicardCommandLine.instanceMain(PicardCommandLine.java:99)
at picard.cmdline.CommandLineProgramTest.runPicardCommandLine(CommandLineProgramTest.java:48)
at picard.analysis.CollectRnaSeqMetricsTest.testNoIntevalsNoFragPercentage(CollectRnaSeqMetricsTest.java:169)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
at org.testng.TestRunner.privateRun(TestRunner.java:782)
at org.testng.TestRunner.run(TestRunner.java:632)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
at org.testng.SuiteRunner.run(SuiteRunner.java:268)
```